src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1024,6 +1096,42 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Private ReadOnly Property Conversion As Conversion
+            Get
+                Return New Conversion(Me.ConversionKind)
```

Comment:
Should not be dropping method symbol information. See how SemanticModel creates Conversion instance for a BoundConversion node.